templates/eap-cd-amq-persistent-s2i.json

```diff
@@ -0,0 +1,924 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,javaee,java,jboss",
+            "version": "1.4.10",
+            "openshift.io/display-name": "JBoss EAP CD + A-MQ (with https)",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example JBoss Enterprise Application Platform continuous delivery application using Red Hat JBoss A-MQ with persistence and secure communication using https. For more information about using this template, see https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/eap-cd/README.adoc",
+            "template.openshift.io/long-description": "This template defines resources needed to develop a JBoss Enterprise Application Platform continuous delivery based application, including a build configuration, application deployment configuration, using Red Hat JBoss A-MQ with persistence and secure communication using https.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
+        },
+        "name": "eap-cd-amq-persistent-s2i"
+    },
+    "labels": {
+        "template": "eap-cd-amq-persistent-s2i",
+        "xpaas": "1.4.10"
+    },
+    "message": "A new JBoss EAP CD and A-MQ persistent based application with SSL support has been created in your project. The username/password for accessing the A-MQ service is ${MQ_USERNAME}/${MQ_PASSWORD}. Please be sure to create the following secrets: \"${HTTPS_SECRET}\" containing the ${HTTPS_KEYSTORE} file used for serving secure content; \"${JGROUPS_ENCRYPT_SECRET}\" containing the ${JGROUPS_ENCRYPT_KEYSTORE} file used for securing JGroups communications.",
+    "parameters": [
+        {
+            "displayName": "Application Name",
+            "description": "The name for the application.",
+            "name": "APPLICATION_NAME",
+            "value": "eap-app",
+            "required": true
+        },
```

Comment:
The 7.1 template also has: https://github.com/jboss-openshift/application-templates/blob/master/eap/eap71-amq-persistent-s2i.json#L31

Reply:
It seems this was removed per a comment from Tomas Remes. So ok. :)

Reply:
Yep, HOSTNAME_HTTP was dropped from the route spec, so then removed as something the user can set. This is a general issue, not just EAP CD.